android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -335,6 +332,46 @@ protected void addUniqueIdentifiers(JSONObject element, String uniqueId) throws
         }
     }
 
+    private void buildRelevanceWithUniqueIds(JSONObject element, String uniqueId) throws JSONException {
+        JSONObject relevance = element.optJSONObject(RELEVANCE);
+        if (relevance != null) {
+            if (relevance.has(RuleConstant.RULES_ENGINE)) {
+                if (widgetArgs != null) {
```

Comment:
Could this `if` statements be merged into one?

Reply:
cool, just separated it since the initial implementation did not require `widgetArgs`